runtime/lib/ttnn/operations/conv/conv2d.cpp

```diff
@@ -31,6 +38,12 @@ void run(const ::tt::target::ttnn::Conv2dOp *op, ProgramContext &context) {
           {op->padding_height(), op->padding_width()},
           {op->dilation_height(), op->dilation_width()}, op->groups(), bias,
           config));
+
+  auto new_memconfig = out.memory_config();
+  new_memconfig.memory_layout = TensorMemoryLayout::INTERLEAVED;
+  new_memconfig.buffer_type = BufferType::DRAM;
+  out = ::ttnn::to_memory_config(out, new_memconfig, std::nullopt);
```

Comment:
This needs an issue tracking it with a comment, ideally nested inside of a block.

Reply:
Do we need new memconfig in case where we didn't override shard_layout above to BS(block sharded :)).